### PR TITLE
Feature | Staff Record | Nursing Questions

### PR DESCRIPTION
**Pull request by ChazUK**
There's a few updates rolled into this PR which is why the files changed is a bit large

- Added Nursing Category Question
- Added Nursing Specialism Question
- Added Nursing Question to Staff Record Employment Summary List
- Added a helper in the Worker Service to check if a Staff Record has a Job Role
- Updated Navigation for first 7 Questions in the Staff Record flow
- Fixed an issue where navigating to a Question from the Staff Record page would not return back on submitting the Question
- Reordered some imports

### CODE REVIEW

**Comment by ChazUK** [comment]
[(5) Enhancement for Navigation on Other Job Role - Front End](https://trello.com/c/yGbtRFJu/52-5-enhancement-for-navigation-on-other-job-role-front-end)

**Comment by ChazUK** [comment]
[(10) Staff Records - 1. Nurse Category question](https://trello.com/c/ZfLqpOJV/50-10-staff-records-1-nurse-category-question)

**Comment by ChazUK** [comment]
[(10) Staff Records - 2. Nurse Specialism question](https://trello.com/c/PkupZnCs/51-10-staff-records-2-nurse-specialism-question)

File: src/app/features/workers/staff-record-summary/staff-record-summary.component.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ export class StaffRecordSummaryComponent {
   }
 
   setReturn() {
+    console.log(this.return);
```
Comment (tgreyuk) [inline src/app/features/workers/staff-record-summary/staff-record-summary.component.ts]: console.log

Comment (ChazUK) [reply src/app/features/workers/staff-record-summary/staff-record-summary.component.ts]: removed

**Pull request merged**